src/ensembl/src/shared/components/paste-or-upload/PasteOrUpload.tsx

```diff
@@ -67,12 +73,12 @@ const PasteOrUpload = (props: PasteOrUploadProps) => {
       {shouldShowInput && (
         <div className={styles.fields}>
           <div className={styles.inputWrapper}>
-            <Input
+            <textarea
```

Comment:
Let's start by creating our own Textarea component (similar to the Input component).

Reply:
Would it not be better to do that in a separate PR?

Reply:
🤷‍♂ 
we can do a separate PR with just the Textarea, which will be considered before this PR. Or include the Textarea component in this PR.

Reply:
I think a separate PR would be better since `Textarea` will be a shared component

Reply:
I am thinking it will be better to create it later on and replace it here so that I can focus more on Custom Downloads tasks.

Reply:
Agreed

Reply:
This PR extracts PasteOrUpload into a shared component. Which means that you think that this component is generic enough to be used somewhere other than in custom download. Which means that you are un-focusing from the purely custom download task.

If you think this is a component that will be usable in other applications, please make it usable for other applications. If not, let it stay in custom download application and extract it to shared component later.

Reply:
I think it is usable for other applications. Please let me know if there is anything that would prevent it from being used elsewhere.

Reply:
> Please let me know if there is anything that would prevent it from being used elsewhere.

Since we don't have any other applications that are currently relying on this component, it's hard to predict what they will need from it. But if we are considering this component to be a standalone shared component, then can we treat it a bit more carefully?

- let's create the Textarea component on which it depends (it will take 10-20 minutes or so)
- let's consider its name —  it is really PasteOrUpload or rather PasteOrReadFromFile?
- let's consider whether it always needs to hide the textarea initially?

Reply:
> let's create the Textarea component on which it depends (it will take 10-20 minutes or so)

Create the component, look for improvements, write tests, add to storybook, create PR, get it reviewed, make changes and so on... No way that's all happening in 20 mins 😄 

> let's consider its name — it is really PasteOrUpload or rather PasteOrReadFromFile?

`ReadFromFile` makes sense as a name but I haven't come across many places where is says 'Read from file' and you are asked to select a file. I named it `PasteOrUpload` because the initial text says "Paste or Upload file".